app/src/main/java/com/SecUpwN/AIMSICD/AIMSICD.java

```diff
@@ -93,7 +93,6 @@
     private ActionBarDrawerToggle mDrawerToggle;
     private CharSequence mDrawerTitle;
     private CharSequence mTitle;
-    public static ProgressBar mProgressBar;
```

Comment:
>It's clear and difficult to break even if violates basic OO principles.

I only agree partially with @agilob. While I think it is OK to keep code that is difficult to break, I feel that it is necessary to follow basic OO principles in order to maintain a higher code quality in the long run. We always should keep in mind that we're working on a security project which any external person is invited to audit. Using hard to break but basic principles ignoreing code may not only knock off developers and security enthusiasts, but also transfers the message that we may not think much about the future: Such code could one day break essential parts of our app.

>SwipeRefreshLayout

Fair. But we need some indicator that there is one. Like [here](https://github.com/Demievil/SwipeRefreshLayout) at the top right corner.